app/Http/Livewire/StreamList.php

```diff
@@ -23,6 +34,12 @@ public function render(): View
             }, function(Builder $builder) {
                 $builder->upcoming()->fromOldestToLatest();
             })
+            ->when($this->search, function(Builder $builder, $search) {
```

Comment:
Is there a way we can get rid of the logic here? Like, put those `when` checks in a scope?